product_docs/docs/pgd/5/planning/limitations.mdx

```diff
@@ -78,9 +78,14 @@ and current listing.
 
 ## Mixed PGD versions
 
-PGD was developed to [enable rolling upgrades of PGD](/pgd/latest/upgrades) by allowing mixed versions of PGD to operate during the upgrade process. 
-We expect users to run mixed versions only during upgrades and, once an upgrade starts, that they complete that upgrade.
-We don't support running mixed versions of PGD except during an upgrade.
+PGD was developed to [enable rolling upgrades of PGD](/pgd/latest/upgrades) by
+allowing mixed versions of PGD to operate during the upgrade process. We expect
+users to run mixed versions only during upgrades. Once an PGD cluster upgrade
+has begun, you should complete the whole cluster upgrade as quickly as possible.
+
+We don't support running mixed versions of PGD except during an upgrade, and we
+don't support clusters running mixed versions, while being upgraded, for a
+extended periods such as over a month.
```

Comment:
While this is sufficiently vague, I think I'd still prefer to not mention any time frame at this point in time (Ribbon may interpret this as a confirmation of their 30 day figure, which I'd like to avoid).

Reply:
```suggestion
extended periods.
```